### PR TITLE
Only show Hub Switcher when there is more than 1 Hub

### PR DESCRIPTION
**Pull request by stonedDiscord**
i really don't need it if i only have 1 hub

### CODE REVIEW

File: server/network/aoprotocol.py
Comment (Crystalwarrior) [inline server/network/aoprotocol.py:298]: These changes are weird, why ignore the area settings? Maybe some servers want only 1 hub and no ARUP (like in my use case for a dr rp server)

**Review by Crystalwarrior — CHANGES_REQUESTED** [review]
Everything good besides over riding ARUP setting

File: server/network/aoprotocol.py
Comment (stonedDiscord) [reply server/network/aoprotocol.py:298]: you'd end up with an empty area list

Comment (stonedDiscord) [reply server/network/aoprotocol.py:298]: addresses

**Comment by stonedDiscord** [comment]
i don't have write access and can't merge it myself,

you have to press merge

**Pull request merged**
